### PR TITLE
docs(play): include billing information in play docs

### PR DESCRIPTION
**Pull request by huygur**
## Description

closes https://github.com/camunda/play/issues/622

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [x] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/components/modeler/web-modeler/play-your-process.md
Comment (huygur) [inline docs/components/modeler/web-modeler/play-your-process.md:204]: Is there a link we can use here to contact CSM? 🤔

File: docs/components/modeler/web-modeler/play-your-process.md
Comment (ultraschuppi) [inline docs/components/modeler/web-modeler/play-your-process.md:202]: our users do not need to know that there is something like an internal plan.

also not sure if trial plan is actually what should be taken? isn't it rather called "Camunda SaaS Trial" by now? probably best for @felix-mueller to comment on

File: docs/components/modeler/web-modeler/play-your-process.md
Comment (ultraschuppi) [reply docs/components/modeler/web-modeler/play-your-process.md:204]: i don't think this is documented anywhere in the outside world

**Review by ultraschuppi — CHANGES_REQUESTED** [review]

File: docs/components/modeler/web-modeler/play-your-process.md
Comment (huygur) [reply docs/components/modeler/web-modeler/play-your-process.md:204]: do you mean we should leave this without a link, or change the wording here

File: docs/components/modeler/web-modeler/play-your-process.md
Comment (huygur) [reply docs/components/modeler/web-modeler/play-your-process.md]: removed the 'Internal Plan', waiting for @felix-mueller suggestion for the Trial 👀

File: docs/components/modeler/web-modeler/play-your-process.md
Comment (ultraschuppi) [reply docs/components/modeler/web-modeler/play-your-process.md:204]: i think leaving it like this is fine - if "CSM" is a term known to customers

**Review by HanselIdes — CHANGES_REQUESTED** [review]

File: docs/components/modeler/web-modeler/play-your-process.md
Comment (huygur) [reply docs/components/modeler/web-modeler/play-your-process.md]: 😮  fixed it

File: docs/components/modeler/web-modeler/play-your-process.md
Comment (huygur) [reply docs/components/modeler/web-modeler/play-your-process.md]: I included this part and added a link to where this is mentioned. please check the plan descriptions again.

I kept the part where enterprise organizations can be marked as 'dev organizations' to avoid costs as @ultraschuppi mentioned this was also a way to avoid the costs.

**Review by HanselIdes — APPROVED** [review]
LGTM

**Review by conceptualshark — CHANGES_REQUESTED** [review]

**Review by conceptualshark — APPROVED** [review]
🚀

**Comment by huygur** [comment]
@conceptualshark the only open question that is blocking this PR to be merge is this : https://github.com/camunda/camunda-docs/pull/4299#discussion_r1758624329 , which was awaiting input from @felix-mueller , but maybe you or @HanselIdes also have information regarding this?

another alternative would be to keep it as it is and release a patch later if @felix-mueller would like it to be changed, as this PR is currently blocking Play to move forward with another feature that requires links to this doc 😅

**Comment by conceptualshark** [comment]
@huygur Unfortunately I don't have any more information on trial plan naming - we use Trial/Free Trial throughout the docs, so if there is a decision to start specifying SaaS (or using another name), we can always make a ticket for that and update it across the docs.

**Comment by huygur** [comment]
@conceptualshark ok then, thanks. I will merge this PR then to unblock Play.

**Pull request merged**
